introduction.md

```diff
@@ -5,3 +5,5 @@ Hi,
 **Name:** Joel
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?